server/models/Fines.js

```diff
@@ -0,0 +1,8 @@
+import { Model } from 'objection';
+import User from '@models/User';
+
+class Fines extends Model {
```

Comment:
All our models are in singular. This should be called `Fine` and not `Fines`. Please let's stick to all conventions so as to maintain a clean codebase.